materials/job_types.md

```diff
@@ -9,7 +9,7 @@ You already got to know the [interactive web interface for Puhti](https://docs.c
 
 Disadvantages of interactive jobs: 
 * Blocks your shell until it finishes
-* Connection interruption means that job is gone
+* Connection interruption means that job is gone (except when running a compute node shell from the web interface)
```

Comment:
```suggestion
* Connection interruption means that job is gone.
     * It is possible to keep job running, while closing the terminal, with persistent compute node shell from the web interface or using [screen](https://www.geeksforgeeks.org/screen-command-in-linux-with-examples/).
```

Reply:
If you mean the normal Computer node shell in web interface, then I'd write it so that it is clear that it is not a different thing, now according to this sentence I'd look for "persistent Compute node shell" .

Reply:
And maybe it should be " a tool called screen" to clarify what it is?

Reply:
Edited for both comments

Reply:
slight rewording